ark/phenotyping/som_utils.py

```diff
@@ -945,9 +971,10 @@ def cluster_cells(base_dir, cluster_counts_name='cluster_counts.feather',
             print(output.strip())
 
 
-def cell_consensus_cluster(base_dir, max_k=20, cap=3, column_prefix='cluster',
+def cell_consensus_cluster(base_dir, cluster_cols, max_k=20, cap=3,
```

Comment:
cell_consensus_cluster should be running on size-normalize data. So should be cell x pixel cluster table that has been normalized by cell size, take the average of the cells in each cluster, then consensus clustering on those averages. Not sure if that's what's happening here.